t/alien.t

```diff
@@ -19,8 +19,21 @@ my $version = $run->out;
 
 my $xpamb_already_running = run_ok( [ qw[ xpaaccess XPAMB:* ] ])->out eq 'yes';
 
+my $win32_processobj;
 unless ( $xpamb_already_running ) {
-    exec( 'xpamb' ) if ! fork;
+    if ($^O eq 'MSWin32') {
+        require Win32::Process;
+        require File::Which;
+        Win32::Process::Create($win32_processobj,
+            File::Which::which("xpamb"),
+            "xpamb",
+            0,
+            32 + 134217728, #NORMAL_PRIORITY_CLASS + CREATE_NO_WINDOW
+            ".") || die $^E;
+        #$pid = $win32_processobj->GetProcessID();
+    } else {
+         exec( 'xpamb' ) if ! fork;
+    }
 
```

Comment:
Your test [log](https://rt.cpan.org/Ticket/Display.html?id=121653) indicates that `xpamb` is running, so the `exec('xpamb')` seems to have worked.  Is `Win32::Process` preferred over fork on Windows?